charts/Geomap.args.js

```diff
@@ -129,15 +129,137 @@ Additionally, a custom formatting function can be passed as a second argument to
+          "geoBottomley",
+          "geoBromley",
+          // "geoChamberlin",
+          "geoChamberlinAfrica",
+          "geoCollignon",
+          "geoCraig",
+          "geoCraster",
+          "geoCylindricalEqualArea",
+          "geoCylindricalStereographic",
+          "geoEckert1",
+          "geoEckert2",
+          "geoEckert3",
+          "geoEckert4",
+          "geoEckert5",
+          "geoEckert6",
+          "geoEisenlohr",
+          "geoFahey",
+          "geoFoucaut",
+          "geoFoucautSinusoidal",
+          "geoGilbert",
+          // "geoGingery",
+          "geoGinzburg4",
+          "geoGinzburg5",
+          "geoGinzburg6",
+          "geoGinzburg8",
+          "geoGinzburg9",
+          "geoGringorten",
+          "geoGuyou",
+          "geoHammer",
+          // "geoHammerRetroazimuthal",
+          // "geoHealpix",
+          "geoHill",
+          "geoHomolosine",
+          "geoHufnagel",
+          "geoHyperelliptical",
+          "geoKavrayskiy7",
+          "geoLagrange",
+          "geoLarrivee",
+          "geoLaskowski",
+          "geoLittrow",
+          "geoLoximuthal",
+          "geoMiller",
+          // "geoModifiedStereographic",
+          "geoModifiedStereographicAlaska",
+          "geoModifiedStereographicGs48",
+          "geoModifiedStereographicGs50",
+          "geoModifiedStereographicMiller",
+          "geoModifiedStereographicLee",
+          "geoMollweide",
+          "geoMtFlatPolarParabolic",
+          "geoMtFlatPolarQuartic",
+          "geoMtFlatPolarSinusoidal",
+          "geoNaturalEarth2",
+          "geoNellHammer",
+          "geoNicolosi",
+          "geoPatterson",
+          "geoPolyconic",
+          "geoRectangularPolyconic",
+          "geoRobinson",
+          "geoSatellite",
+          "geoSinusoidal",
+          "geoSinuMollweide",
+          "geoTimes",
+          // "geoTwoPointAzimuthal",
+          "geoTwoPointEquidistantUsa",
+          "geoVanDerGrinten",
+          "geoVanDerGrinten2",
+          "geoVanDerGrinten3",
+          "geoVanDerGrinten4",
+          "geoWagner",
+          "geoWagner4",
+          "geoWagner6",
+          "geoWagner7",
+          "geoWiechel",
+          "geoWinkel3",
+          // "geoInterruptedHomolosine",
+          // "geoInterruptedSinusoidal",
+          // "geoInterruptedBoggs",
+          // "geoInterruptedSinuMollweide",
+          // "geoInterruptedMollweide",
+          // "geoInterruptedMollweideHemispheres",
+          // "geoInterruptedQuarticAuthalic",
+          "geoPolyhedralButterfly",
+          "geoPolyhedralCollignon",
+          "geoPolyhedralWaterman",
+          "geoGringortenQuincuncial",
+          "geoPeirceQuincuncial",
+          // "conicConformalFrance",
+          // "conicConformalPortugal",
+          // "conicConformalSpain",
+          // "onicConformalEurope",
+          // "conicEquidistantJapan",
+          // "mercatorEcuador",
+          // "transverseMercatorChile"
+        ]
+      },
```

Comment:
```js
import d3Geo from "d3-geo";

const excludedKeys = ["conicConformalFrance", "conicConformalPortugal", ...];

const options = Object.keys(d3Geo).
  .filter(d => !excludedKeys.includes(d));
```

Reply:
🧠  I extended it to exclude "Raw" projections and excludedKeys as you suggested:
`Object.keys(projections).filter(d => (!d.includes("Raw")) && (!excludedKeys.includes(d)))`